query-coordinator/src/main/scala/com/socrata/querycoordinator/QueryParser.scala

```diff
@@ -288,4 +229,63 @@ object QueryParser {
   private val andFn = SoQLFunctions.And.monomorphic.get
 
   private val systemColumns = Set(":id", ":created_at", ":updated_at", ":version").map(ColumnName(_))
+
```

Comment:
Yes, just a move from instance based class to singleton object so that I can use this somewhere else like a global function.  I was thinking about making this its own PR/commit to reduce total change in complicated PR.